packages/components/src/Notification/GlobalNotification/_docs/GlobalNotification.stickersheet.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import {
+  GlobalNotificationProps,
+  GlobalNotification,
+} from "../GlobalNotification"
+
+export default {
+  title: "KAIO-staging/Notifications/Global Notification",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Looks like axe is having issues with the overlapping elements in stickersheets causing false positives.
+            id: "color-contrast",
+            enabled: false,
+          },
+        ],
+      },
```

Comment:
Hmm I don't think this should be an issue for you, since we found it has to do with reverse stickersheets. Or is there another reason this is here?

Reply:
Happy to remove, wasn't sure if this was just an issue with testing the animate in for non persistent notification

Reply:
Can confirm not an issue on local, so will push up the change 👍

Reply:
hmmm actually may need to add that timeout back in since the test failed, will rerun to confirm